templates/github-automated-workflow/workflow.yaml

```diff
@@ -0,0 +1,33 @@
+name: __NAME__
+
+github:
+  # https://docs.github.com/en/github/searching-for-information-on-github/searching-for-repositories
+  #    all modules
+  #    repo_search: "org:cloudposse"
+  #    terraform modules
+  repository_search: "repo:cloudposse/terraform-aws-sns-topic"
+
+  branch: auto-__NAME__
+  commit_message: auto-__NAME__
+  pr:
+    enabled: true
+    body:
```

Comment:
Convert to string. We can use `|-` to make it multiline but this frees us up to use any syntax